.github/workflows/pytest.yml

```
@@ -10,14 +10,18 @@ jobs:
   test:
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        python-version: [3.10, 3.11, 3.12]
```

I think some checks are failing because versions should be strings:
```suggestion
        python-version: ["3.10", "3.11", "3.12"]
```

---

I've committed your suggestion ✅